src/Servers/IIS/IIS/src/Core/IISServerAuthenticationHandler.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNetCore.Server.IIS.Core
     /// <summary>
     /// The default authentication handler with IIS In-Process
     /// </summary>
-    public class IISServerAuthenticationHandler : IAuthenticationHandler
+    internal class IISServerAuthenticationHandler : IAuthenticationHandler
```

Comment:
Found a couple public projects that used this type, not sure how important it is to keep it, but just an FYI
https://github.com/halllo/IdentityServerExperiments/blob/267d6ba82a217d23e9fa966ad74505b717c42c28/IdentityServer/Startup.cs#L116
https://github.com/RegionOrebroLan/.NET-Web-Authentication-Extensions/blob/c96f94abc54df06ccaa8bf5f815225b3f95eb152/Source/Project/Extensions/TypeExtension.cs#L52

Reply:
That first one seems like a very outdated workaround to set the DisplayName. I don't quite follow what that second one is doing.